### PR TITLE
[WIP] Rockplanet Season Pass 2: Frying the Burrito.

### PR DESCRIPTION
**Pull request by ghost**
### Planned content includes:

- [ ] Functional weather, Dust Storms: relatively non-lethal, but you don't want to be caught unprepared in one.
- TODO: make this not a subtype of ashstorms
- TODO: add sprites
- TODO: stat tweaking to be less-lethal (50 cap on brute, maybe eye damage if there's no goggles/mask?)
- [ ] Unique flora, 3 to be specific: Coyote Tobacco, Dagger-Plant Yucca, and Wine-colored Agaricus mushrooms.
- TODO: testing and stat tweaking
- TODO: yucca and agaricus need full implementation(?)
- [ ] Unique mobs, based around STALKER and PREY.
- TODO: all of it lmao
- [x] New and Updated ruins using said assets.
- TODO: testing and stat tweaking
- [x] A whole load of custom assets for ruins and mapping. Not just for Rockplanet, but for anywhere that needs 'em!
- TODO: testing and stat tweaking
- [ ] Redone aesthetic, including duller rocks and new ambient noise, and random clutter spawning.
- TODO: all of it whoops
- [ ] New ghost role, the Scavenger/Salvage Guild, subsidiary of the Free Trade Union. 
- NOTE: Rockplanets' version of the ash tribe(?), their contention point with the miners is, rather than defending tendrils, they loot ruins + future plans. 
- TODO: How will this be fully implemented? IDK yet but the guild is part of a future plan too so either way at least their stuff is being packed in here. Might just make a lone scav as the rockplanet version of the pod crash.
- TODO: decide if I want it to be named the scav or salvage guild. probably scav for the shorthand name.

**Credit to most sprites goes to Fortune13 and STALKER13 - I've modified practically all of them to fit better, but the credit is still due. I'm no spriter, and having the sprites either as a base or just as-is has helped make this possible.**

```
Stuff to look for in Pass 3:
- Unique Megafauna
- Unique gamer-loot
- Other, more anomalous stuff :)
```

## Why It's Good For The Game
Rockplanet my beloved, you deserve this tender loving and care.

## Changelog
:cl:
expansion: Nanotrasen Terraforming Division is proud to announce that they've restarted the atmospheric cycle of LV-669! Sadly, due to the lack of water, all this has resulted in is terrible dust storms. However, with the sands shifting, more of the past has been uncovered, good and bad.
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
GET OUT OF HERE STALKER

**Comment by jjpark-kb** [comment]
since its WIP, put it into draft (ill do it for you this time)

**Comment by ghost** [comment]
> since its WIP, put it into draft (ill do it for you this time)

I thought it did it automatically im sorry aaaaa
(Also this is my mobile account just pretend it says orion)

**Comment by KasparoVy** [comment]
I really love your vision for the rock planet! Good luck STALKER

**Comment by ghost** [comment]
ok so its up to date but TOO much up to date, aaaaghhgbhgb

File: tgstation.dme [hunk 1]
```diff
@@ -4648,13 +4649,9 @@
 #include "modular_skyrat\modules\QOL\code\_under.dm"
 #include "modular_skyrat\modules\radiosound\code\game\objects\items\devices\radio\headset.dm"
 #include "modular_skyrat\modules\random_mining\code\mining_maps.dm"
```
Comment (ghost) [inline tgstation.dme]:
```suggestion
#include "modular_skyrat\modules\random_mining\code\mining_maps.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\anvil.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\forge.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\forge_clothing.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\forge_items.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\forge_recipes.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\forge_weapons.dm"
#include "modular_skyrat\modules\reagent_forging\code\modules\reagent_forging\water_basin.dm"
```
Dont know why these were deleted by the pr, but this should fix it

**Comment by ghost** [comment]
and bam. Now up-to-date, with my changes kept.

Its 330 am this was annoying.

**Comment by ghost** [comment]
OK
YKNOW WHAT

SHITS FUCKED
IM MAKING A NEW BRANCH, CLOSING THIS PR. NEW ONE WILL OPEN EVENTUALLY.

**Pull request closed without merging**